Doc/library/http.cookies.rst

```diff
@@ -294,3 +296,26 @@ The following example demonstrates how to use the :mod:`http.cookies` module.
    >>> print(C)
    Set-Cookie: number=7
    Set-Cookie: string=seven
+
+
+Handling of Duplicate Cookies
+-----------------------------
+
+As per ``RFC 6265``, the ``http.cookies`` module has been updated to better align with standard practices for handling duplicate cookies. Previously, if multiple cookies with the same name were encountered, the last value provided would be retained. With the update, the first value encountered for a given cookie name is preserved, reflecting the behavior commonly expected by web servers and user agents.
```

Comment:
Instead of ``` ``RFC 6265`` ```, please, use `rfc` directive ([example](https://github.com/python/cpython/blob/87faec28c78f6fa8eaaebbd1ababf687c7508e71/Doc/library/http.cookies.rst?plain=1#L20))
Same for ``` ``http.cookies`` ``` ([example](https://github.com/python/cpython/blob/87faec28c78f6fa8eaaebbd1ababf687c7508e71/Doc/library/http.cookies.rst?plain=1#L14))